src/main/java/com/streisky/discordchatgptbot/command/Command.java

```diff
@@ -1,13 +1,17 @@
 package com.streisky.discordchatgptbot.command;
 
+import com.streisky.discordchatgptbot.command.animegirl.AnimeGirlCommand;
+import com.streisky.discordchatgptbot.command.chatgpt.ChatGPTCommand;
+import com.streisky.discordchatgptbot.command.ping.PingCommand;
 import com.streisky.discordchatgptbot.exception.InvalidCommandException;
 
 import java.util.Arrays;
 
 public enum Command {
 
-    PING("ping", new PingPongCommand()),
-    CHATGPT("chatgpt", new ChatGPTCommand());
+    PING("ping", new PingCommand()),
+    CHATGPT("chatgpt", new ChatGPTCommand()),
+    ANIME_GIRL("animegirl", new AnimeGirlCommand());
```

Comment:
For every new command class, a new enum will have to be created. Maybe this could belong to the implementation of the command class itself.

Another thing that may happen when using the enum is to make dependency injection more difficult.

Reply:
Not worth it for this project.